### ISSUE TITLE
[iOS] Expense Report.json: 'Export As PDF' link is not accessible with Voiceover using Swipe gestures in forward and backward both navigation.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6418]
### Target Platforms

iOS

### SDK Version

App Version: Version 1.0 (2.3.1-beta.20210823.1)

### Application Name

Visualizer

### Problem Description

[ADO 35693107](https://microsoft.visualstudio.com/OS/_workitems/edit/35693107)

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.2" control and select it.
4. Navigate on  "V1.2" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "Expense Report.json" control and select it.
7. Navigate the 'Expense Report.json' screen and enable Voiceover for the ease of access.
8. Try to navigate to 'Export as PDF' link using swipe gestures and observe the issue.

**Actual Result:**
'Export As PDF' link is not accessible with Voiceover using Swipe gestures in forward and backward both navigation. present under 'ExpenseReport.json' card.

**Expected Result:**
'Export As PDF' link should be made accessible with Voiceover using Swipe gestures in forward and backward both navigation. present under 'ExpenseReport.json' card.

### Card JSON

```JSON
ExpenseReport.json
```

**Comment by jwoo-msft** [comment]
not reproducible in the current build

**Comment by vagpt** [comment]
Working fine on the current build.
